apps/dialtone-documentation/docs/design/colors/usage/index.md

```diff
@@ -0,0 +1,234 @@
+---
+title: Usage
+description: A systematic and accessible color palette that supports both functional needs and personalized experiences within our products.
+---
+
+<div class="d-m32"></div>
+
+## Overview
+
+Dialtone's color system offers a consistent visual foundation with a brand purple, versatile grayscale, and semantic status colors. Subtle gradients are used for promotional accents.
+
+What we prioritize is semantic clarity, ensuring each color serves a distinct purpose within the UI for clear communication and consistency...
+
+<div class="d-m32"></div>
+
+<div class="d-bgc-purple-400 d-bgo10 d-bar12 ">
+ <svg-loader class="d-fl1" name="color-table" />
+ </div>
+
+### Brand Colors
+
+  Brand colors are non-negotiable for immediate product recognition. Each Dialpad product is visually defined by its unique brand color.
+
+### Neutral Colors
+
+Blacks, whites, and grays—form the bedrock of our interfaces. They dictate surfaces, text, and layout, often signaling state changes within components. Lighter neutrals strategically guide the eye, establishing a clear visual hierarchy
+
+### Semantic Colors
+
+Semantic colors are our messengers, conveying feedback, status, and urgency. They are fundamental to Dialtone's token system, ensuring that critical information is always visually prioritized.
```

Comment:
I'm not sure this is adding any value. A few options: rewrite, or move their lines closer to the associated color they address, or perhaps remove and revisit later.